pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[build-system]
+requires = ["setuptools>=63.0"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+platforms = ["Linux x86, x86-64"]
+
+[tool.setuptools.packages.find]
+include = ["sopel", "sopel.*"]
+namespaces = false
+
+[project]
+name = "sopel"
+version = "8.0.0.dev0"
+description = "Simple and extensible IRC bot"
+authors = [
+  { name="dgw", email="dgw@technobabbl.es" },
+]
```

Comment:
Should list more than just me. Many people were responsible for creating the code that eventually became Sopel, then for maintaining Sopel before I took over as maintainer, and now for helping me maintain it (including you).

Reply:
Let's make this its own issue, but not necessarily for the 8.0.0 milestone if how to structure the metadata about past contributors isn't clear yet.

Reply:
Done in #2339. Just the new comments from @SnoopJeDi left now.